src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -434,13 +434,15 @@ public Task<bool> RecordLatestMatureDepositsAsync(IList<MaturedBlockDepositsMode
                                 CrossChainTransferStatus status = CrossChainTransferStatus.Suspended;
                                 Script scriptPubKey = BitcoinAddress.Create(deposit.TargetAddress, this.network).ScriptPubKey;
 
-                                if (!haveSuspendedTransfers)
+                            if (!haveSuspendedTransfers)
+                            {
+                                var fixedTransferCost = new Money(0.01m, MoneyUnit.BTC);
```

Comment:
is this some randmon number? it should be a const

Reply:
It will be. Was focusing on the concept and first.